tchannel/messages/common.py

```diff
@@ -66,6 +67,22 @@
 )
 
 
+def random_tracing():
+    """
+    Create new Tracing() tuple with random IDs.
+    """
+
+    def _uniq_id():  # TODO: cython
+        """Create a random 64-bit unsigned long."""
+        return random.getrandbits(64)
```

Comment:
This probably doesn't affect performance significantly, but it's not a common pattern in this codebase. The leading `_` in the name should be enough in discouraging its use from elsewhere. Please move it outside.